src/components/seances/SeanceCard.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+  <div class="seance-card">
+    <div class="seance-card__poster">
+      <img
+        :src="
+          $store.getters.movieList.filter((movie) => movie.id == movieId)[0]
+            .poster_url
+        "
+        alt="poster"
+      />
+    </div>
+    <div class="seance-card__details">
+      <h3>
+        {{
+          $store.getters.movieList.filter((movie) => movie.id == movieId)[0]
+            .title
+        }}
+      </h3>
+      <div class="seance-card__details__info">
+        <span class="seance-card__details__info__genre">
+          {{
+            $store.getters.movieList.filter((movie) => movie.id == movieId)[0]
+              .genre.name
+          }}
+        </span>
+        <span class="seance-card__details__info__time">
+          {{
+            $store.getters.movieList.filter((movie) => movie.id == movieId)[0]
+              .length
+          }}
+          min
+        </span>
+      </div>
+      <ul class="seance-card__details__list">
+        <li
+          v-for="seance in seancesByMovie"
+          :key="`seance_${seance.id}`"
+          class="seance-card__details__list__item"
+        >
+          {{ seance.datetime.substring(11, 16) }}
```

Comment:
it's hard for me to guess what is the output of this, maybe it's worth to create a method that has some semantic name?